ios/ZBarSDK/ZBarHelpController.h

```diff
@@ -36,12 +37,12 @@
 // failure dialog w/a few useful tips
 
 @interface ZBarHelpController : UIViewController
-                              < UIWebViewDelegate,
+                              < WKNavigationDelegate,
```

Comment:
This will be a breaking change, since this will only work for MABS 6+. Are all aware of that?

Reply:
It is not a breaking change. This has nothing to do with our webview. This is another view controller that shows a WKWebView to present an HTML (help screen). We don't use the exposed delegate in our code.

Reply:
Yes, you are right! I thought this was the plugin... shame!

Reply:
My only fear regarding breaking changes is MABS 3.3, because the lib was built in Xcode 10 and not 9. But the previous version has no issue for them, so...

Reply:
Yep, but MABS 3 is no longer supported. This new version will be only compatible with MABS 5 and 6.